src/interface/web/app/home/page.tsx

```diff
@@ -0,0 +1,463 @@
+'use client'
+
+import styles from './chat.module.css';
+import React, { Suspense, useEffect, useState, useMemo } from 'react';
+
+import SuggestionCard from '../components/suggestions/suggestionCard';
+import SidePanel from '../components/sidePanel/chatHistorySidePanel';
+import NavMenu from '../components/navMenu/navMenu';
+import Loading from '../components/loading/loading';
+import useSWR from 'swr';
+import Image from 'next/image';
+
+import 'katex/dist/katex.min.css';
+
+import { StreamMessage } from '../components/chatMessage/chatMessage';
+import ChatInputArea, { ChatOptions } from '../components/chatInputArea/chatInputArea';
+import { useAuthenticatedData } from '../common/auth';
+
+//samples for suggestion cards (should be moved to json later)
+const suggestions: Suggestion[] = [["Automation", "blue", "/automate.svg", "Send me a summary of HackerNews every morning.", "/automations?subject=Summarizing%20Top%20Headlines%20from%20HackerNews&query=Summarize%20the%20top%20headlines%20on%20HackerNews&crontime=00%207%20*%20*%20*"], ["Automation", "blue", "/automate.svg", "Compose a bedtime story that a five-year-old might enjoy.", "/automations?subject=Daily%20Bedtime%20Story&query=Compose%20a%20bedtime%20story%20that%20a%20five-year-old%20might%20enjoy.%20It%20should%20not%20exceed%20five%20paragraphs.%20Appeal%20to%20the%20imagination%2C%20but%20weave%20in%20learnings.&crontime=0%2021%20*%20*%20*"], ["Paint", "green", "/paint.svg", "Paint a picture of a sunset but it's made of stained glass tiles", ""], ["Online Search", "yellow", "/online_search.svg", "Search for the best attractions in Austria Hungary", ""]];
```

Comment:
These .svg files shouldn't be referenced here. The suggestion cards should just use the icons from phosphoricons.